### DISCUSSION TITLE
Some problem with Magick.Native-Q16-HDRI-x64.dll

### DISCUSSION THREAD

**Discussion by Plamonster** [dlemstra/Magick.Native#18] [Q&A]
Hi!
Now that I have successfully built both Magick.Native and .Net (see yesterday discussion https://github.com/dlemstra/Magick.Native/discussions/16), I'm not able to use them in my solution :(
I got an exception thrown by 
               var imageInfo = new MagickImageInfo(wSourcePath);
The inner exception is :
_innerException = {"Unable to load DLL 'Magick.Native-Q16-HDRI-x64.dll': The specified module could not be found. (Exception from HRESULT: 0x8007007E)"}

I tried to add Magick.Native-Q16-HDRI-x64.dll as a reference in my solution, but I can't as it is a C++ dll. 
I see in the Magick.Net MagickImage class that a bunch of DLLImport are used. 
I manually copied the Magick.Native-Q16-HDRI-x64.dll file from my <Magick.Native.Folder>\src\Magick.Native\bin\ReleaseQ16-HDRI\x64\ to the <Magick.NET.Fodler>\src\Magick.Native\libraries\win. 
Is this correct?
Any Hint?

**Comment by dlemstra** [comment]
I am sorry but I cannot assist you with this. You will need to check the build scripts in my projects and try to replicate that.

**Reply by Plamonster** [reply]
I added the <Magick.NET.Fodler>\src\Magick.Native\libraries\win in my path and it fixes the problem.
Merci!
